App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/ManageJob/ManageJob.jsx

```diff
@@ -74,10 +111,106 @@ export default class ManageJob extends React.Component {
         });
     }
 
+    handleFilterChange(event, data) {
+        var value = data.value;
+
+        var newFilter = TalentUtil.deepCopy(this.state.filter);
+
+        Object.keys(newFilter).forEach(v => {
+            newFilter[v] = value.includes(v);
+        });
+        
+        this.loadNewData({ filter: newFilter, activePage: 1 });
+    }
+
+    handleSortChange(e, { value }) {
+        this.loadNewData({ sortBy: { date: value }, activePage: 1 });
+    };
+
+    handlePageChange(e, { activePage }) {
+        if (activePage != this.state.activePage) {
+            this.loadNewData({ activePage: activePage });
+        }
+    }
+
     render() {
+        var filterOptions = [
+            { key: 'showActive', text: 'Show Active', value: 'showActive' },
+            { key: 'showClosed', text: 'Show Closed', value: 'showClosed' },
+            { key: 'showDraft', text: 'Show Draft', value: 'showDraft' },
+            { key: 'showExpired', text: 'Show Expired', value: 'showExpired' },
+            { key: 'showUnexpired', text: 'Show Unexpired', value: 'showUnexpired' }
+        ]
+
+        var sortOptions = [
+            { key: 'desc', text: 'Newest first', value: 'desc' },
+            { key: 'asc', text: 'Oldest first', value: 'asc' }
+        ]
```

Comment:
Are these variables changing? If not, we can utilise **const** instead of **var**. Then if it is changing we can use **let** so you can familiarise with ES6 format. Also, don't forget to close your variable initialisation using **;.** in the end of your code